### DISCUSSION TITLE
大佬，有考虑过支持多个qb下载器吗？

### DISCUSSION THREAD

**Discussion by WPF0414** [jxxghp/MoviePilot#430] [建议]
目前一个是正常给mp用的，一个用来刷流，又想使用mp的播放限速来解决在播放时处理上传限制，不知大佬可有加入多个下载器的想法？

**Comment by mjsoftking** [comment]
QB下载，TR辅种就行

**Reply by WPF0414** [reply]
目前是这样，不过我还有个专门用来刷流的qb，相当于三个下载器。

**Reply by WPF0414** [reply]
刷流的资源是不想转移到辅种里去的，当然刷流也不是必须的，只是咨询一下会不会像nt一样支持多个下载器

**Reply by mjsoftking** [reply]
以后加不加看大佬的安排了

不过没必要弄3个吧，2个够用，MP转移做种的就把不想做种的放到其他目录下并把这个目录加到不转移列表，IYUU转移的就把需要转移的加入转移列表

**Reply by mjsoftking** [reply]
如果是多个盘用，那就映射多个下载目录，目前就是不支持刷流和自由下载目录设置

**Reply by WPF0414** [reply]
有道理 我去试试看 谢谢

**Reply by WPF0414** [reply]
突然想起来刷流是有删种规则的，如果共用可能会把正常订阅下载的给删了。

**Reply by mjsoftking** [reply]
其实也有办法的，就是把路径作为关键字加到保存路径关键词
